build/nodes-and-endpoints/join-testnet.md

```diff
@@ -335,10 +335,49 @@ git checkout $BINARY_VERSION
 make build
 cp build/persistenceCore ~/.persistenceCore/cosmovisor/upgrades/$BINARY_VERSION/bin
 ```
-
 Now, at the upgrade height, **Cosmovisor** will upgrade swap the binaries.
 
-### Manual Option
+**2. Auto-download option**   
+Above manual option, still needs upgrade version binary to be manually added under cosmovisor/upgrades/upgrade-name/bin directory. 
```

Comment:
The above manual option still needs an upgrade version binary to be manually added under cosmovisor/upgrades/upgrade-name/bin directory.

Reply:
noted